src/main/io/gps.c

```diff
@@ -917,7 +939,9 @@ enum {
 } ubs_nav_fix_type;
 
 enum {
-    NAV_STATUS_FIX_VALID = 1
+    NAV_STATUS_FIX_VALID = 1,
+	NAV_STATUS_TIME_WEEK_VALID = 2,
```

Comment:
Please set your editor to use 4 spaces instead of tabs, and replace the tabs with 4 spaces where they have slipped in here.